### PR TITLE
Better naming for Crowdin workflows

### PR DESCRIPTION
**Pull request by lkiesow**
Let all GitHub Actions workflow definitions for Crowdin…

- have a filename starting with `crowdin-…`
- Have a name starting with `Crowdin » `

### CODE REVIEW

**Review by ziegenberg — APPROVED** [review]
Looks good to me.

**Review by ziegenberg — CHANGES_REQUESTED** [review]
The line
```
crowdin download --config .crowdin.yaml -b admin-ui-picard
```
probably also needs changing, and the branch name updated to `main`.

That's what the last GHA error suggest: https://github.com/opencast/opencast-admin-interface/actions/runs/9153874134/job/25163529960

**Comment by lkiesow** [comment]
Fixed conflicts and branch name

File: .github/workflows/crowdin-download-translations.yml [hunk 1]
```diff
@@ -16,26 +16,31 @@ jobs:
...
       - name: add new translations
         run: |
           git add /src/i18n/org/opencastproject/adminui/languages/
+
       - name: update language list
         working-directory: /src/i18n/org/opencastproject/adminui/languages/
         run: |
           echo -n '[ "' > locales.json
           echo -n ??-??.json | sed 's/  */", "/g' >> locales.json
           echo '" ]' >> locales.json
```
Comment (ziegenberg) [inline .github/workflows/crowdin-download-translations.yml:40]: Do we need this step? Isn't the list of languages configured here `src/i18n/languages.ts`?

**Review by ziegenberg — CHANGES_REQUESTED** [review]

File: .github/workflows/crowdin-download-translations.yml [hunk 2]
```diff
@@ -16,26 +16,31 @@ jobs:
...
       - name: download translations
         env:
           CROWDIN_TOKEN: ${{ secrets.CROWDIN_TOKEN }}
         run: |
-          crowdin download --config .crowdin.yaml -b admin-ui-picard
+          crowdin download --config .crowdin.yaml -b main
+
       - name: add new translations
         run: |
           git add /src/i18n/org/opencastproject/adminui/languages/
```
Comment (ziegenberg) [inline .github/workflows/crowdin-download-translations.yml:33]: Should we have those absolute file paths here? Same with the next step

Comment (gregorydlogan) [reply .github/workflows/crowdin-download-translations.yml:33]: The upstream project uses `git commit -a` here.  Unsure which is less safe :D

Comment (ziegenberg) [reply .github/workflows/crowdin-download-translations.yml:33]: I'm just not sure if an absolute file path would work or if this should be relative to the current working directory...

Comment (ziegenberg) [reply .github/workflows/crowdin-download-translations.yml:33]: @lkiesow, could you quickly look at this? Then, we could merge it right after.

**Review by ziegenberg — APPROVED** [review]

**Comment by ziegenberg** [comment]
Looking good now!

**Pull request merged**